src/bundles/robot_simulation/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Robot simulator for EV3.
+ *
+ * @module robot_simulation
+ * @author Joel Chan
+ */
+
+export {
+  createCustomPhysics,

```

Comment:
Are we using pascal case naming for these exported functions? @RichDom2185

Reply:
Oh, I think you meant camelCase not PascalCase, but yes, you're right. Though I don't think we it should block the PR (since it's only Part 1 anyway).

@JoelChanZhiYang in your subsequent parts can you convert the exported functions to snake_case to match the rest of our modules?

Reply:
On second thought, I realise EV3 functions are written as `ev3_getColorSensor` ("camel-case"-like style), so please KIV on this.